tasks.py

```diff
@@ -44,6 +44,13 @@ def tests(c):
     c.run(f"{VENV_BIN}/pytest", pty=PTY)
 
 
+@task
+def coverage(c):
+    """Generate code coverage of running the test suite"""
```

Comment:
Sorry! One more thing… I think the linter will complain (not yet, but at some point in the future) without a period at the end of the doc-string, so perhaps get ahead of that and add it now?

Reply:
Added, just to get this merged.

But, I wouldn't bother with stuff like that. If the linter isn't complaining, leave it.

Then, fix the linter, and fix all the stuff at once. Every other task in this file doesn't have a period, so there will be a lot of stuff to fix.

Reply:
Fair point, and duly noted 😊